fixd.js

```diff
@@ -56,7 +57,9 @@ const fixd = {
     if (typeof modifier !== 'function') {
       throw new Error('Modifier must be a function')
     }
-    return fixd.freeze(modifier(clone(fixd[name].$fixdVal)))
+    const newObj = modifier(clone(fixd[name].$fixdVal))
+    if (newObj) return fixd.freeze(newObj)
+    throw new Error('Modifier should return a value')
```

Comment:
`if (newObj)` doesn't account for falsy values, which may be valid returns.  Suggest checking for nil (null/undefined) explicitly.